policy_tests/Makefile

```diff
@@ -47,6 +47,19 @@ bare-qemu_TIMEOUT = 360
 bare-qemu: CONFIG=bare-qemu
 bare-qemu: all
 
+# config for 64-bit bare-metal on QEMU
+bare64-qemu_SIM = qemu
+bare64-qemu_TESTS = bare64
+bare64-qemu_RUNTIME = bare64
+bare64-qemu_MODULE = osv.bare64.main
```

Comment:
The module name shouldn't have the architecture in it, since the policies themselves are architecure-independent.

Reply:
Resolved by runtime/arch split.